rep-0003.rst

```diff
@@ -124,6 +124,31 @@ Indigo Igloo (May 2014)
   - releasing, documenting and continuous integration testing only catkin-based packages
 - rosbuild-based packages can still be built from source
 
+Jade Turtle (May 2015 - May 2017)
```

Comment:
I've added a commit which points out that the start and end dates for distribution support is outlined in the already linked to http://wiki.ros.org/Distributions/Timeline wiki page.

We can do further upgrades to that page for clarification and to define the exact meaning of EOL for a ROS distro, but I think this is within the scope of this REP and supported by external links to documentation.